docs/recipes/precompiling-with-webpack.md

```diff
@@ -7,20 +7,26 @@ The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) ment
 ###### webpack.config.js
 
 ```js
+const path = require('path');
+// To pre-compile folder with multiple test files, we can use glob
+// see http://stackoverflow.com/questions/32874025/how-to-add-wildcard-mapping-in-entry-of-webpack/34545812#34545812
+// const glob = require('glob');
+
 const nodeExternals = require('webpack-node-externals');
 
 module.exports = {
 	entry: ['src/tests.js'],
+	// entry: glob.sync('./test/**/*.js'),
```

Comment:
Not exactly.. It is the combined time for execution in 3 node envs.. But the test duration did go down from nearly 13 mins of 100% resource usage to under 3 mins in one run. 

Do you think it would be more appropriate for this recipe to discuss multiple test files transplantion  by default? I understand it can also be used for cases where you have aliases too. But to me it seems like the former is the more prominent use case.